<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="extractLayout">
      <item>
       <widget class="QLabel" name="labelExtract">
        <property name="text">
         <string>Extract to: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditExtract"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseExtract">
        <property name="statusTip">
         <string>Select folder to extract</string>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="listFiles"/>
    </item>
    <item>
     <widget class="Line" name="divider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="saveToLayout">
      <item>
       <widget class="QLabel" name="labelSaveTo">
        <property name="text">
         <string>Save to: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSaveTo"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseSaveTo">
        <property name="statusTip">
         <string>Select folder to save</string>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="extAndResetlayout">
      <item>
       <widget class="QPushButton" name="extractButton">
        <property name="statusTip">
         <string>Start Extract</string>
        </property>
        <property name="text">
         <string>Extract</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="extractProgress">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuWindow">
     <property name="title">
      <string>Window</string>
     </property>
     <addaction name="actionView_statusbar"/>
     <addaction name="actionView_toolbar"/>
    </widget>
    <addaction name="menuWindow"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_use"/>
    <addaction name="actionSee_the_license"/>
    <addaction name="actionAbout_Creator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHow_to_use"/>
   <addaction name="actionResetWin"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/icon/img/exit.png</normaloff>:/icon/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHow_to_use">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/icon/img/help.png</normaloff>:/icon/img/help.png</iconset>
   </property>
   <property name="text">
    <string>How to use</string>
   </property>
   <property name="statusTip">
    <string>See how to use application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSee_the_license">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/icon/img/license.png</normaloff>:/icon/img/license.png</iconset>
   </property>
   <property name="text">
    <string>See the license</string>
   </property>
   <property name="statusTip">
    <string>See the license</string>
   </property>
  </action>
  <action name="actionAbout_Creator">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/icon/img/about_creator.png</normaloff>:/icon/img/about_creator.png</iconset>
   </property>
   <property name="text">
    <string>About Creator</string>
   </property>
   <property name="statusTip">
    <string>About Creator</string>
   </property>
  </action>
  <action name="actionView_statusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/icon/img/statusbar.png</normaloff>:/icon/img/statusbar.png</iconset>
   </property>
   <property name="text">
    <string>View statusbar</string>
   </property>
  </action>
  <action name="actionView_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/icon/img/toolbar.png</normaloff>:/icon/img/toolbar.png</iconset>
   </property>
   <property name="text">
    <string>View toolbar</string>
   </property>
  </action>
  <action name="actionReset_Window">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/icon/img/reset_window.png</normaloff>:/icon/img/reset_window.png</iconset>
   </property>
   <property name="text">
    <string>Reset Window</string>
   </property>
   <property name="statusTip">
    <string>Reset Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionResetWin">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/icon/img/reset_window.png</normaloff>:/icon/img/reset_window.png</iconset>
   </property>
   <property name="text">
    <string>Reset Window</string>
   </property>
   <property name="statusTip">
    <string>Reset Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections/>
</ui>
